docs/source/example_notebooks/prediction/dowhy_causal_prediction_demo.ipynb

```diff
@@ -0,0 +1,1526 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "7a84e574",
+   "metadata": {},
+   "source": [
+    "# Demo for DoWhy Causal Prediction on MNIST\n",
+    "\n",
+    "We're adding prediction functionality to DoWhy. The goal of this notebook is to demonstrate an example of causal prediction using *Causally Adaptive Constraint Minimization (CACM)* [1]. \n",
```

Comment:
can remove the first sentence: we're adding.. to dowhy"

Reply:
add a link to CACM. And then move all references to the bottom of the notebook. This is to avoid interrupting the flow